src/adapters/index.ts

```diff
@@ -302,3 +315,18 @@ export const supportedProtocols: Record<
     [Chain.Linea]: [XfaiDexAdapter],
   },
 }
+
+export const WriteActionInputs = {
+  AaveV3ATokenWriteActionInputs,
+  CompoundV2BorrowMarketWriteActionInputs,
+  CompoundV2SupplyMarketWriteActionInputs,
+}
+
+export const GetTransactionParamsInputSchema = z.union([
+  AaveV3ATokenGetTxParamsInput,
+  CompoundV2BorrowMarketGetTxParamsInput,
+  CompoundV2SupplyMarketGetTxParamsInput,
+])
+export type GetTransactionParamsInput = z.infer<
+  typeof GetTransactionParamsInputSchema
+>
```

Comment:
This code is automatically generated when calling `npm run build-types`.

It'll be based on the exports from the adapter.